src/Frontend/Core/Header/Header.php

```diff
@@ -140,7 +140,7 @@ public function __construct(KernelInterface $kernel)
         $this->addJS('/src/Frontend/Core/Js/utils.js', true, true, Priority::core());
         $this->addJS('/src/Frontend/Core/Js/frontend.js', true, true, Priority::core());
         // @custom for SumoCoders
-        $this->addJS('/src/Frontend/Core/Js/bundle.js', false, true, Priority::core());
+        $this->addJS('/src/Frontend/Core/Layout/bundle.js', false, true, Priority::core());
```

Comment:
Is dat geen vreemde locatie?

Reply:
Goh, niet direct een zotte mening. Vond het vreemd, maar met uw uitleg erbij kan ik er op zich wel mee leven.